recipes/supertree/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+Copyright (c) MLJAR sp. z o. o.
```

Comment:
All the license files are in the tarball. You don't need to add them here again, just list them in the recipe and conda-build will do the right thing.

Reply:
Thank you. I removed license files. I don't know where to put licenses list in the recipe, something like `license_file: LICENSE.txt super tree-non-commercial-license.pdf supertree-commercial-license.pdf`?

Reply:
This file is still here.

Reply:
oh, got it. thank you!